css/style.css

```diff
@@ -107,7 +115,359 @@
+}
+
+.tonic-header .canopies {
+  width: 55px;
+  height: 16px;
+
+  /* Caption / 1 - Bolder */
+  font-family: 'Poppins', 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  color: #344563;
+  list-style: none;
+}
+
+.tonic-header .canopy-list .canopies {
+  margin-top: 10px;
+}
+
+.tonic-header .back {
+  /* Text */
+  margin-top: 10px;
+  font-family: 'Poppins', 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  color: #7a869a;
+}
+
+.tonic-header .canopy-list {
+  display: flex;
+  flex-direction: row;
+  gap: 40px;
+}
+
+.tonic h1 {
+  /* Project Title */
+
+  width: 295px;
+  height: 36px;
+  font-family: 'Poppins', 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  color: #172b4d;
+}
+
+.Multi-Post h1 {
+  width: 295px;
+  height: 25px;
+  font-family: 'Poppins', 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 24px;
+  color: #172b4d;
+}
+
+.tonic-body {
+  /* Primary Text */
+
+  width: 295px;
+  height: 76px;
+  font-family: 'Poppins', 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+
+  /* or 160% */
+
+  /* Neutrals/N600 */
+  color: #344563;
+
+  /* Inside auto layout */
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.work-category {
+  margin-top: 20px;
+}
+
+.work-category ul {
+  list-style: none;
+}
+
+.work-category-language li p {
+  float: left;
+  font-family: 'Poppins', 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+
+  /* Primary/B400 */
+  color: #6070ff;
+}
+
+.Projects {
+  margin-top: 20px;
+}
+
+.Projects button {
+  width: 150px;
+  color: #396df2;
+  background-color: white;
+  border: 1px solid #396df2;
+  padding: 10px;
+  border-radius: 4%;
+  text-align: center;
+  font-family: 'Poppins', 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  font-style: normal;
+  font-size: 17px;
+  font-weight: bold;
+}
+
+.resume button {
+  width: auto;
+  color: #396df2;
+  background-color: white;
+  border: 1px solid #396df2;
+  padding: 10px;
+  border-radius: 5%;
+  text-align: center;
+  font-family: 'Poppins', 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  font-style: normal;
+  font-size: 17px;
+}
+
+.Projects button:hover {
+  border: 1px solid #396df2;
+  padding: 15px;
+}
```

Comment:
- [ ] Kindly make sure all your project buttons (**`See Project`** buttons and the   **`See Project`**) follow the button styling: **Active**, **Hover** and **Pressed** (see design). Check the right **colors** and **other properties** from Figma design on the similar image below:
![buttons](https://user-images.githubusercontent.com/83042742/170831776-ae8cdcc3-84db-4887-a897-78dfa2b08b10.png)